lib/seattleflu/id3c/cli/command/etl/fhir.py

```diff
@@ -553,3 +558,29 @@ def observation_resource(device: str) -> Any:
             )
         )
     }
+
+
+def create_redcap_uri(record: REDCapRecord) -> str:
+    """
+    Create a JSON data URI representing the source of a given REDCap *record*.
+
+    Used in FHIR Encounter resources as the meta.source, which ultimately winds
+    up in ID3C's ``warehouse.encounter.details`` column.
+    """
+    data_scheme = 'data:application/json'
+
+    redcap_data = {
+        'url': 'https://redcap.iths.org',
```

Comment:
This is a hideously tedious point (software 😑), but all the DETs we receive and our DET ETL routines use a URL with a trailing slash (`https://redcap.iths.org/`). I wonder if we should match that here to prevent future potential hair-pulling?

…

Ahhh, and looking now, we could get that value without hardcoding it here by using `record.project.base_url`.

Reply:
AH right, I'll avoid hardcoding the URL here.